### PR TITLE
Confirm Password

### PR DESCRIPTION
**Pull request by Kodylow**
Adds a confirm password form field, we used to have one but now with the generate password feature it's more important that we make sure they actually have the correct password saved.

## Summary by CodeRabbit

- **New Features**
  - Added a confirm password field in the configuration settings.
  - Enhanced form validation to include password confirmation.
  - Introduced error messaging for password mismatches.

### CODE REVIEW

File: apps/guardian-ui/src/components/SetConfiguration.tsx [hunk 1]
```diff
@@ -279,6 +282,22 @@ export const SetConfiguration: React.FC<Props> = ({ next }: Props) => {
...
+                onChange={(ev) => setConfirmPassword(ev.currentTarget.value)}
+                placeholder='Confirm Password'
+              />
+              {password !== confirmPassword && (
+                <FormHelperText color='red'>
+                  {t('set-config.error-password-mismatch')}
+                </FormHelperText>
+              )}
+            </FormControl>
+          )}
```
Comment (elsirion) [inline apps/guardian-ui/src/components/SetConfiguration.tsx:300]: Wait, if this is on the same screen won't people just copy/paste it? Should probably be a "surprise" when they try to click continue.

Comment (Kodylow) [reply apps/guardian-ui/src/components/SetConfiguration.tsx:300]: kk I can do that give me a sec

Comment (Kodylow) [reply apps/guardian-ui/src/components/SetConfiguration.tsx:300]: image

Confirm password pops as a modal when you click next then you have to verify it

**Comment by Kodylow** [comment]
closing since I'd PRed from my master by accident, reopening on branch

**Pull request closed without merging**